src/PruneUnusedImagesModal.jsx

```diff
@@ -103,7 +103,7 @@ const PruneUnusedImagesModal = ({ close, unusedImages, onAddNotification, users
                     <ImageOptions key={user.name}
                                   images={unusedImages.filter(image => image.uid === user.uid)}
                                   name={"deleteImages-" + user.name}
-                                  checked={deleteOwners.includes(user)}
+                                  checked={deleteOwners.some(u => u.uid === user.uid)}
```

Comment:
Seems a bit odd -- the objects in `users` shouldn't change while the dialog is open. But it doesn't hurt indeed.

Reply:
I agree it's weird but it flakes reproducible in the test yet I can't easily reproduce it in a normal dev env.